sheets/1/exercise-3.tex

```diff
@@ -1,150 +1,170 @@
+Sie werden genutzt, um anderen (aber auch euch selbst) kenntlich zu machen, was im Code passiert, was noch verändert werden sollte oder um allgemein Anmerkungen zu hinterlassen.
+
+\begin{lstlisting}[title=\textbf{Kommentar Beispiel}]
+// Dies ist ein einfacher Kommentar über eine Zeile.
+// Er wird mit // gestartet und nimmt die gesamte Zeile hinter sich ein.
+someCode.run() //Kommentare können auch nach Operationen geschrieben werden.
+/*
+ * Kommentare können sich auch über mehrere Zeilen erstrecken.
+ * Diese beginnen mit /* und enden mit  */ . 
+ * Um die Lesbarkeit zu verbessern, werden die Zeilen dazwischen meist 
+ * mit einem * begonnen.
  */
-        \end{lstlisting}
-          In dieser Aufgabe sind alle in < > geschriebenen Wörter Platzhalter für das was sie beschreiben. Jeder Platzhalter kann mit etwas anderem ausgefüllt werden, auch wenn sie gleich heißen. Die Klammern <> müssen dabei mit ersetzt werden!
-    \item \textbf{[1]} Eine Klasse hat immer am Anfang das Schlüsselwort \lstinline{class}, gefolgt von einem frei wählbaren Namen. Darauf folgen die \{ \}, die den gesamten Inhalt der Klasse umschließen.
-          Zum Inhalt einer Klasse gehören \textbf{[1]}Attribute und Operationen
-
-          \textbf{[1]} Eine Operation hat immer einen frei wählbaren Namen, vor dem ein sogenannter Rückgabetyp steht.
-          Einen Rückgabetyp kann mit einem Klassennamen angeben oder mit \lstinline{void}, was gleich bedeutend mit keinem      Rückgabetyp ist.
-          Eine Operation sollte einen Sichtbarkeits-Modifier haben.
-          Was ein Modifier genau macht wird noch erklärt, deshalb verwenden wir einfach mal immer \lstinline{public}.
-          Alle Modifier kommen noch vor den Rückgabetypen.
-          Auf der anderen Seiten vom Namen müssen \lstinline{()} stehen.
-          In diesen können Parameter stehen. Damit verlangt die Operation nach zusätzliche Informationen, die sie braucht. Ein Parameter besteht immer aus einem Typ gefolgt von einem Namen.
-          Beim Typ verhält es sich genau so wie beim Rückgabetyp vorhin. Nur das hier \lstinline{void} nicht erlaubt ist.
-          Sollte man mehrere Parameter brauchen, kann man diese mit einem \lstinline{,} trennen.
-          Nach den \lstinline{()} folgen \texttt{ \{\} }, die einen Codeblock umschließen. Darin stehen alle Anweisungen, die in den Folgebeispielen genauer erläutert werden.
-
-          \begin{lstlisting}[title={\textbf{Klassen Syntax}}]
+\end{lstlisting}
+\lstset{
+	basicstyle=\small
+}
+\subsubsection*{Klassen:}
+Klassen sind Baupläne für Objekte. In diesen können Operationen und Attribute definiert werden, welche dann den Objekten zur Verfügung stehen.\\
+Eine Klasse beginnt immer mit dem \lstinline{class} Schlüsselwort, gefolgt von ihrem Namen. Diesen könnt ihr fast frei wählen, nur spezielle Schlüsselwörter wie zum Beispiel \lstinline{class} sind verboten. Anschließend folgt ein Paar geschweifter Klammern \{ \}, in denen der Inhalt der Klasse geschrieben wird.
```

Comment:
Bitte jeden Satz in eine Eigene Zeile im sourcecode machen.
Am besten noch erwähnen, dass der Klassenname exakt mit dem Dateinamen übereinstimmen muss (dank eclipse).